configuration/nginx/https.conf

```diff
@@ -5,6 +5,7 @@ upstream django {
 
 uwsgi_cache_path /var/cache/nginx/feeds keys_zone=feeds_cache:10m max_size=10g
                  inactive=10m use_temp_path=off;
+limit_req_zone $binary_remote_addr zone=adminlimit:10m rate=1r/s;
```

Comment:
sorry, I'll ask you to set this as `5r/m` because I think it makes more sense and can prevent even abusive scanners that implement sleeps between requests. Then I'll merge it